doc/ml_features.rst

```diff
@@ -0,0 +1,341 @@
+###########
+ML Features
+###########
+
+In this section, we illustrate features which
+users are expected to use in HPC workloads, especially when
+simulation and AI are required to interact. The topics are
+explained through code snippets,
+with code that goes beyond SmartSim and SmartRedis API
+(e.g. code showing how to jit-script a PyTorch model): the
+intention is that of showing *one* simple way of leveraging
+a feature, but more optimized ways of using third-party
+libraries may exist.
+
+Examples are written in Python, but the same
+result can be achieved with any SmartRedis client (C, C++,
+Fortran and Python). Please refer to SmartRedis API
+for language-specific details.
+
+ML Model Deployment and Execution in the Database
+===================================================
+
+The combination of SmartSim and SmartRedis allows users
+to store more than simple tensors on the DataBase (DB). In the following
```

Comment:
Is there are a reason "DataBase" is capitalized like that?  Shouldn't it just be database?